src/category_theory/preadditive/opposite.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import category_theory.preadditive
+import data.equiv.transfer_instance
+
+/-!
+# If `C` is preadditive, `Cᵒᵖ` has a natural preadditive structure.
+
+-/
+
+open opposite
+
+namespace category_theory
+
+variables (C : Type*) [category C] [preadditive C]
+
+instance : preadditive Cᵒᵖ :=
+{ hom_group := λ X Y, equiv.add_comm_group (op_equiv X Y),
```

Comment:
This is transferring data to the new typeclass: so I think there should be some (possibly simp) lemmas describing it

Reply:
Thanks, yes. I just noticed this in #7525. I'll backport now.